### DISCUSSION TITLE
What are the five colors of the Olympic rings?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#405] [Galaxy Trivia] [answered]
*Give colors in #RRGGBB format*

**Comment by thuongtruong1009** [comment, marked as answer]
The colors of the Olympic rings in #RRGGBB format are:

Blue: #007FFF
Yellow: #FFD700
Black: #000000
Green: #008000
Red: #FF0000
Note that the colors of the rings may vary slightly in different contexts and representations, but these are the most commonly accepted values for the official Olympic colors.

**Comment by peterbaccaro** [comment]
The official HEX codes for the Olympic ring colors are:

BLUE: **#0078D0**
YELLOW: **#FFB114**
BLACK: **#000000**
GREEN: **#00A651**
RED: **#F0282D**

These HEX codes are based on the Olympic logo from the official Olympic website (https://olympics.com/) and then using a color picker to reveal the HEX codes for each of the colors.  Are you impressed? :smiley: :ok_hand: :thumbsup:

**Reply by ParanoidUser** [reply]
Actually, I am. Thank you for taking the extra step to specify the colors. 😅
